components/mdx/Code.tsx

```diff
@@ -1,192 +1,173 @@
 // @ts-nocheck
 
 import React from "react";
-import Highlight, { defaultProps } from "prism-react-renderer";
-import { Prism } from "prism-react-renderer";
-import rangeParser from "parse-numeric-range";
-import classnames from "classnames";
+import Highlight, { Prism, defaultProps } from "prism-react-renderer";
+import classNames from "classnames";
 
 import FileIcon from "../icons/File";
 import CopyIcon from "../icons/Copy";
 import CopyButton from "../Copy";
-import classNames from "classnames";
 
 // Extend base classes
-(typeof global !== "undefined" ? global : window).Prism = Prism;
-require("prismjs/components/prism-docker");
+(typeof global === "undefined" ? window : global).Prism = Prism;
```

Comment:
Can't we use globalThis for this?

Reply:
Yes! of course

Reply:
I'm so sad, it built fine for dev but it looks like it didn't for realsies

Reply:
Oh, this isnt' the problem, globalThis works, the static import does not as it is called before executing code. The dynamic import seems to be working with no issues so I'll revert that. I suspect because the code is completely synchronous and everything to do with prism is done much later anyways. I hope I'm not shooting my foot here, but we can always ignore that line and switch back to require if need be.